src/app/models/evt-models.ts

```diff
@@ -127,6 +127,23 @@ export interface WitnessGroup {
     groupId: string;
 }
 
+export type ApparatusEntriesList = Map<ApparatusEntry>;
+
+export interface ApparatusEntry extends GenericElementData {
+    id: string;
+    content: ApparatusReading[];
```

Comment:
I would distinguish lemma from other readings

Reply:
I think it's not necessary, because lem and rdg are all readings.  I made this distinction through the `tagname` property in the ApparatusReading interface.

Reply:
@ChiaraDipi @szenzaro Is it ok if I create an ApparatusLemma interface for lemma and an ApparatusReading interface for other readings? Or maybe it's better to group everything in "ApparatusReading" and create new fields? I didn't understand how you imagined it.

Reply:
You can just call the interfaces "Lemma" and "Reading" ;)

Reply:
Done